assets/dev/js/editor/document/hooks/ui/settings/change-post-title.js

```diff
@@ -0,0 +1,27 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class ChangePostTitle extends After {
```

Comment:
There is no reason for adding a whole new hook file for each setting change. Add one file that listens to document settings change and accordingly applys the callbacks.

Reply:
it's a separated feature it's not a part of how the editor should act.

Reply:
So do one hook handler for all changes that need external handling.